rancher/en/securing-rancher/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Securing Rancher
+layout: bpg-default-v1.0
+version: v1.0
+lang: en
+---
+
+## Securing Rancher
+---
```

Comment:
This document is titled "Securing Rancher," but other than the first section and possibly the second, it has nothing to do with Rancher. It moves into "general security practices for Docker," and I think that we should limit the BPG to clear, Rancher-specific guidelines. I didn't think about this until I got to the end, but I wanted to make a note of it somewhere.